timeseries/src/autogluon/timeseries/models/gluonts/abstract_gluonts.py

```diff
@@ -310,12 +311,13 @@ def _fit(
             dataset=train_data, callbacks=self._get_callbacks(time_limit=time_limit), **kwargs
         )
 
-        estimator = self._get_estimator()
-        with warning_filter(), disable_root_logger(), gluonts.core.settings.let(gluonts.env.env, use_tqdm=False):
-            self.gts_predictor = estimator.train(
-                self._to_gluonts_dataset(train_data),
-                validation_data=self._to_gluonts_dataset(val_data),
-            )
+        with TemporaryDirectory() as temp_dir:
+            estimator = self._get_estimator(root_dir=temp_dir)
```

Comment:
I understand the tempdir is creeping all the way up here (for all gluonts models instead of just torch) because the train call is here. Is there another way around this?

Reply:
Thanks for the comments, I agree will all of them. Now the `lightning_logs` are saved to the model directory and are deleted after the training is finished (we don't use the checkpoint, GluonTS gives us the trained model's weights). Does this look better to you?